src/extension.ts

```diff
@@ -1006,14 +1014,17 @@ export async function activate(context: vscode.ExtensionContext) {
             const examplesFolder = await vscode.window.showQuickPick(
               [
                 {
+                  description: "ESP-IDF",
```

Comment:
I would suggest based on these `espIdfPath`, `espAdfPath` and `espMdfPath` we can just disable these options or not show them at all? What do you think?
Also if we just have only one available then just short circuit this step completely!

Reply:
But they are available ! If you had set ESP-MDF or ESP-ADF (with commands or manually set configuration settings) you can see their examples rendered here.